content/docs/intro/cloud-providers/okta/setup.md

```diff
@@ -12,20 +12,20 @@ The [Pulumi Okta provider]({{< relref "./" >}}) uses the Okta SDK to manage reso
 
 Once obtained, there are two ways to communicate your configuration tokens to Pulumi:
 
-1. Set the environment variables `OKTA_ORG_NAME`, `OKTA_BASE_URL` and `OKTA_API_KEY`:
+1. Set the environment variables `OKTA_ORG_NAME`, `OKTA_BASE_URL` and `OKTA_API_TOKEN`:
 
-    ```bash
+    ```console
```

Comment:
Can we keep these as `bash`? I'm not sure if the styling will recognize `console`. In addition, the copy button won't properly strip out the prompts unless it's a recognized (https://github.com/pulumi/docs/blob/94c6cfcef5a80b857937b261f7b3fb3855e4aa70/assets/js/copybutton.js)